packages/now-client/src/create-deployment.ts

```diff
@@ -68,19 +68,19 @@ export default function buildCreateDeployment(
     // Get .nowignore
     let ig = await getNowIgnore(path);
```

Comment:
```suggestion
    let { ig, ignores } = await getNowIgnore(path);
```

maybe this would make the rest a bit more readable